extensions-android/README.md

```diff
@@ -33,9 +33,9 @@ Android.
 Pre-condition:
 
   Update crosswalk version in `xwalk-echo-extension-src/build.xml` before run `build.sh`.<br />
-  e.g. change stable 8.37.189.14 to canary 15.44.375.0
+  e.g. change stable 8.37.189.14 to canary 18.46.452.0
```

Comment:
We can change the version code by auto, but it's difficult for canary/beta/stable.